docs/resources/built-in-rules.md

```diff
@@ -328,3 +328,225 @@ Here is the equivalent of the `recommended` configuration values:
 
 OpenAPI-cli supports [custom rules](./custom-rules.md).
 However, if you have an idea for a built-in rule you believe will benefit the greater API community, please [open an issue](https://github.com/Redocly/openapi-cli/issues/new).
+
+## Generic built-in rules
+
+You can use special `enforcements` rule for cases which our other built-in rules do not cover.
+There are [a set of generic built-in rules](#list-of-generic-built-in-rules) which can be configured under `enforcements` to enforce different generic usecases.
+
+### How it works?
+
+Let's say you want to validate Operation summary and Info description fields - they should be defined, not empty, 
+minimum 30 chars length and both have "_hello_" substring inside.
```

Comment:
I would use some more realistic example here. `_hello_` substring is not a realistic use case. How about "should end with a full stop"?

Reply:
Then, nonEmpty and minimum 30 chars are redundant. You need only 30 chars.